src/Microsoft.AspNet.Mvc.Core/Rendering/Expressions/ExpressionMetadataProvider.cs

```diff
@@ -55,11 +56,11 @@ public static class ExpressionMetadataProvider
             }
 
             var container = viewData.Model;
-            Func<object> modelAccessor = () =>
+            Func<object, object> modelAccessor = (c) =>
```

Comment:
`container` variable is no longer used; so you can use `container` as the lambda parameter